commands/info.go

```diff
@@ -21,47 +21,101 @@ var (
 	}
 )
 
+// infoArguments represents the arguments we can make use of in this command
+type infoArguments struct {
+	token       string
+	verbose     bool
+	apiEndpoint string
+}
+
+// defaultInfoArguments returns an infoArguments object populated by the user's
+// command line arguments
+func defaultInfoArguments() infoArguments {
+	return infoArguments{
+		token:       cmdToken,
+		verbose:     cmdVerbose,
+		apiEndpoint: cmdAPIEndpoint,
+	}
+}
```

Comment:
The actual reason is that it simplifies testing. Instead of setting a bunch of global variables between several test cases, this allows to set the arguments specificly for one test case, not affecting other tests.

It's also an alignment with other commands, as I have already started introducing this in previous PRs.

I just haven't rewritten the tests yet.